tests/omprog-restart-terminated-outfile.sh

```diff
@@ -71,7 +71,7 @@ injectmsg 6 1
 injectmsg 7 1
 wait_queueempty
 
-kill -s USR1 $(get_child_pid)
+kill -s KILL $(get_child_pid)
 ./msleep 100
```

Comment:
We should try to avoid sleep calls - past months experience tells they work exceptionally bad when CI is pretty busy. We have lot's of failures because of this. Again, this is nothing specific to the PR, just sharing info.

Reply:
I know sleeps are not desirable... When I refactored the omprog tests in #2629 I eliminated most of them, and as far as I know the tests have been stable since then (please tell me otherwise). However I left that one (which ensures that the child processes the signal before receiving the next message) to keep the test code simpler, and because it seems very unlikely that the test could break because of this... Taking note of this as a possible future improvement, though.